pyrsa/data/exampledataset.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
```

Comment:
How is this going to be different from Dataset/DatasetBase? Currently it looks like a duplicate

Reply:
The Dataset class is the default/vanilla DatasetBase child that takes in a standard input and perform standard operations. However, the DatasetBase parent enables customized inputs and operations, as in exampledataset.py and neurodataset.py. The user should be able to simply use Dataset for their standard input without further coding, while more advanced user might wish to build upon the DatasetBase for their customized data operations.

Reply:
Yes, that is all clear, but what will exampledataset do? 
Is this an example how a user would define their own dataset? Or what is this specific file for?

Reply:
Yes, it is an example how a user would define their own dataset. For users that are not expert programmers, it would be good to have an example for them to build on, where they can just implement the part of preprocess function.